### PR TITLE
[Network-Manager] Migration to CodeGen

### PR DESCRIPTION
**Pull request by calvinhzy**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Network-Manager

**Comment by kairu-ms** [comment]
You can remove this long option name 
image

File: src/network-manager/azext_network_manager/custom.py [hunk 1]
```diff
@@ -47,19 +21,21 @@ def network_manager_create(client,
...
     parameters['id'] = id_
     parameters['location'] = location
     parameters['tags'] = tags
     parameters['description'] = description
     parameters['network_manager_scopes'] = network_manager_scopes
     parameters['network_manager_scope_accesses'] = network_manager_scope_accesses
-    return client.create_or_update(resource_group_name=resource_group_name,
-                                   network_manager_name=network_manager_name,
-                                   parameters=parameters)
+    return Create(parameters)
```
Comment (kairu-ms) [inline src/network-manager/azext_network_manager/custom.py:35]: Please use this way
```suggestion
    return _NetworkManagerCreate(cli_ctx=cmd.cli_ctx)(command_args=parameters)
```

File: src/network-manager/azext_network_manager/custom.py [hunk 2]
```diff
@@ -68,206 +44,27 @@ def network_manager_update(instance,
...
-                                       regions=None):
-    client = cf_networkmanagementclient(cmd.cli_ctx)
-    parameters = {}
-    parameters['skip_token'] = skip_token
-    parameters['regions'] = regions
-    return client.list_active_connectivity_configurations(resource_group_name=resource_group_name,
-                                                          network_manager_name=network_manager_name,
-                                                          parameters=parameters)
+        parameters['network_manager_scope_accesses'] = network_manager_scope_accesses
+    return Update(parameters)
```
Comment (kairu-ms) [inline src/network-manager/azext_network_manager/custom.py:64]: Same as above

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**